### PR TITLE
Add new switch info for build script

### PR DESCRIPTION
**Pull request by drewgillies**
...Including the new SkipDevX switches.

### CODE REVIEW

**Comment by drewgillies** [comment]
@alpaix Are you happy with this?

File: README.md [hunk 1]
```diff
@@ -62,6 +62,19 @@ A [full list of all the repos](https://github.com/NuGet) is available as well.
 
 ######In case you have build issues please clean the local repo using `git clean -xdf` and retry building
 
+####Notable .\build.ps1 switches
+- `-SkipDev14` - builds only binaries targeting Visual Studio 15 (Dev15)
+- `-SkipDev15` - builds only binaries targeting Visual Studio 2015 (Dev14)
+
+Note that if only one of Visual Studio 2015 (Dev14) or Visual Studio 15 (Dev15) is installed, neither of the above switches is necessary - the script will build according to tyhe installed version.
```
Comment (alpaix) [inline README.md]: typo in _tyhe_

**Comment by drewgillies** [comment]
@alpaix Fixed.

**Comment by alpaix** [comment]
:ok: for command line options.
Probably need another sign-off by a native speaker :smiley_cat:

**Comment by emgarten** [comment]
:shipit:

**Comment by drewgillies** [comment]
Thanks!

**Pull request merged**